hoomd/md/pair/alch.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2009-2021 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause
+# License.
+
+"""Alchemical potentials."""
+
+from hoomd.logging import log, Loggable
+from hoomd.operation import _HOOMDBaseObject
+from hoomd.data.parameterdicts import TypeParameterDict
+from hoomd.data.typeconverter import SetOnce
+from hoomd.md.pair import pair
+
+# alternative access? hoomd.alch.md.pair would be nice to somehow link here
+# without making overly complicated
+# I just like alch(MD/MC) as alchemy specification aliases
```

Comment:
I think we'd want to leave things in specialized locations unless something like `import hoomd.alch` was called to make easier access.

Reply:
We will move all alchemy related potentials and methods to a new `hoomd.md.alchemy` namespace.